apps/hash-api/src/graph/util.ts

```diff
@@ -128,6 +128,10 @@ const getOrCreateOwningAccountGroupId = async (
 export const ensureAccountGroupOrgsExist = async (params: {
   context: ImpureGraphContext;
 }) => {
+  if (isSelfHostedInstance) {
+    return;
+  }
```

Comment:
The `@linear` org is to host system types, which are external if it's self-hosted.

Hosted instance:
- create `@linear` types and org if integration is enabled (saves us seeding the types now since we're going to update them)

Self-hosted instance:
- never create any orgs for system types

Reply:
Alright, I just thought we need the organization regardless if the integration is enabled. Even better that way! 🎉